public/src/components/channelManagement/bannerTests/bannerTestEditor.tsx

```diff
@@ -135,6 +136,35 @@ const BannerTestEditor: React.FC<ValidatedTestEditorProps<BannerTest>> = ({
     updateTest({ ...test, locations: updatedRegions });
   };
 
+  function concatUniqueLocations(countryGroups: Region[], locations: Region[]) {
```

Comment:
I don't think this is necessary if we initialise `regionTargeting` with the country groups from `locations`.
Here's a proposal for doing this which I think simplifies things a lot - what do you think? -
https://github.com/guardian/support-admin-console/pull/682

Reply:
This is great, I am happy to merge this